rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerEventReceiver.java

```diff
@@ -101,6 +102,11 @@ private void openAndReceive(final SchedulerEventListener eventListener, boolean
         socket = client.create();
         EventNotificationHandler notificationHandler = new EventNotificationHandler(eventListener);
         socket.on(Event.MESSAGE, notificationHandler);
+        socket.on(Event.CLOSE, new Function() {
+            public void on(Object t) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1161)